src/qt/trafficgraphwidget.cpp

```diff
@@ -11,18 +11,20 @@
 #include <QColor>
 #include <QTimer>
 
+#include <chrono>
 #include <cmath>
 
 #define DESIRED_SAMPLES         800
 
 #define XMARGIN                 10
 #define YMARGIN                 10
 
+using namespace std::chrono_literals;
+
```

Comment:
Why this change?

Reply:
I checked through the file and made sure that this statement is not needed.

Removed in f7a19ef774ef92ce348215593e3590a750c345e1